api/graphql/mutations/group.js

```diff
@@ -139,7 +138,7 @@ export async function inviteGroupToGroup(userId, fromId, toId, type, questionAns
   }
 
   // If current user is a moderator of both the from group and the to group they can automatically join the groups together
-  if (await GroupMembership.hasModeratorRole(userId, toGroup)) {
+  if (await GroupMembership.hasModeratorRole(userId, toGroup, {}, Responsibility.constants.RESP_ADMINISTRATION)) {
```

Comment:
yeah seems like this could be the default, though probably better to actually make it so you have to pass in an responsibility always, so its explicit